core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -4491,6 +4491,21 @@ private void checkNullOperand(SqlTester tester, String op) {
         "null", "VARCHAR(2000) NOT NULL");
   }
 
+  @Test public void testJsonStorageSize() {
+    tester.checkString("json_storage_size('[100, \"sakila\", [1, 3, 5], 425.05]')",
+            "29", "INTEGER");
+    tester.checkString("json_storage_size('{\"a\": 1000, \"b\": \"a\", \"c\": \"[1, 3, 5, 7]\"}')",
+            "37", "INTEGER");
+    tester.checkString("json_storage_size('{\"a\": 1000, \"b\": \"wxyz\", \"c\": \"[1, 3]\"}')",
+            "34", "INTEGER");
+    tester.checkString("json_storage_size('[100, \"json\", [[10, 20, 30], 3, 5], 425.05]')",
+            "36", "INTEGER");
+    tester.checkString("json_storage_size('null')",
+            null, "INTEGER");
```

Comment:
Can you explain this one? 'null' is a string, why does it return null? Is there case for ` It returns NULL if the argument is NULL.`?

Reply:
well,Thanks for reminding me. I adjusted this mistake.